README.md

```diff
@@ -91,8 +153,26 @@ end
 ```
 
 - What is mystery5("hi")?
+>mystery5("hi") = string of two asterisks  
```

Comment:
oh yes! I couldn't figure out how to put in strings of asterisks into the markdown file -- the formatting was going wild.